brian2/devices/cpp_standalone/templates/main.cpp

```diff
@@ -17,6 +17,8 @@
 
 #include <iostream>
 #include <fstream>
+#include <sstream>
+#include <string>
```

Comment:
I think `sstream` was only needed for your `to_str` function, right?
```suggestion
#include <string>
```

Reply:
yes